cloud/pkg/cloudstream/iptables/iptables.go

```diff
@@ -45,14 +43,15 @@ type iptablesJumpChain struct {
 const (
 	// the tunnelPort chain
 	tunnelPortChain utiliptables.Chain = "TUNNEL-PORT"
+	streamPort                         = 10003
```

Comment:
Hard-coding here, maybe needs to dynamically query it from the cloudcore configmap while the goroutine is woking up every time.

Reply:
Let's use cloudStream.port here :)